src/travel-aid/TravelAidScreenComponent.tsx

```diff
@@ -0,0 +1,154 @@
+import {Close} from '@atb/assets/svg/mono-icons/actions';
+import {Button} from '@atb/components/button';
+import {EstimatedCallInfo} from '@atb/components/estimated-call';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {ServiceJourneyDeparture} from '@atb/travel-details-screens/types';
+import React from 'react';
+import {SafeAreaView} from 'react-native-safe-area-context';
+import {useTravelAidDataQuery} from './use-travel-aid-data';
+import {ScrollView} from 'react-native-gesture-handler';
+import {GenericSectionItem, Section} from '@atb/components/sections';
+import {View} from 'react-native';
+import {ThemeText} from '@atb/components/text';
+import {
+  formatToClock,
+  formatToClockOrRelativeMinutes,
+  formatToISODate,
+} from '@atb/utils/date';
+import {dictionary, useTranslation} from '@atb/translations';
+import {TravelAidTexts} from '@atb/translations/screens/subscreens/TravelAid';
+import {ThemeIcon} from '@atb/components/theme-icon';
+import {Realtime as RealtimeDark} from '@atb/assets/svg/color/icons/status/dark';
+import {Realtime as RealtimeLight} from '@atb/assets/svg/color/icons/status/light';
+import {StatusBarOnFocus} from '@atb/components/status-bar-on-focus';
+import {MessageInfoBox} from '@atb/components/message-info-box';
+
+export type TravelAidScreenParams = {
+  serviceJourneyDeparture: ServiceJourneyDeparture;
+};
+type Props = TravelAidScreenParams & {
+  goBack: () => void;
+};
+export const TravelAidScreenComponent = ({
+  serviceJourneyDeparture,
+  goBack,
+}: Props) => {
+  const styles = useStyles();
+  const {language, t} = useTranslation();
+  const {themeName} = useTheme();
+
+  const {data: serviceJourney} = useTravelAidDataQuery(
+    serviceJourneyDeparture.serviceJourneyId,
+    formatToISODate(serviceJourneyDeparture.serviceDate),
+  );
+
+  // TODO: Change focused stop over time
```

Comment:
Since we have a TODO on this, maybe add one TODO regarding error handling and service journey not found?